homeassistant/components/mopeka/config_flow.py

```diff
@@ -69,7 +102,8 @@ async def async_step_user(
             await self.async_set_unique_id(address, raise_on_progress=False)
             self._abort_if_unique_id_configured()
             return self.async_create_entry(
-                title=self._discovered_devices[address], data={}
+                title=self._discovered_devices[address],
+                data={DATA_MEDIUM_TYPE: user_input[USER_INPUT_MEDIUM_TYPE]},
```

Comment:
```suggestion
                data={CONF_MEDIUM_TYPE: user_input[USER_INPUT_MEDIUM_TYPE]},
```

Reply:
@bdraco Can you plesae clarify.  Previously data={DATA_MEDIUM_TYPE: user_input[USER_INPUT_MEDIUM_TYPE]}, had the same keys.  In your review you said they were not the same so I changed them to be different.  It appears your new comment suggest a name change but to make them the same.  Personally, I kind of feel these are just keys and if they happen to be the same in both dictionaries that's OK but I'm happy to go either way.  thanks

Reply:
For clarity, my concern was about mixing the step names and the data.  There is no concern about using the same keys for the user data and the config entry data since they are ultimately the same data.